docs/3_manage/clear_index.md

```diff
@@ -9,8 +9,7 @@ es_clear_index -[options]...
 
 Both of the scripts have the same usage and behavior.
 
-```
-Usage: ruby [type]_clear_index.rb [collection] -[options]...
+```(type)_clear_index -[options]...
```

Comment:
Do you think it would be clearer to use `(es|solr)` for these? It doesn't explicitly say those are the "type" values above the example lines with `(type)`.

Reply:
I'll switch them

Reply:
Okay, thanks. I guess if we added or expected to add more types, then maybe just listing them explicitly as "type" values would be cleaner, but with our short list for now this should be manageable. :crossed_fingers: